[2022-07-13 09:09:15,428] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v3.second_task manual__2022-07-13T09:09:13.019343+00:00 [queued]>
[2022-07-13 09:09:15,468] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v3.second_task manual__2022-07-13T09:09:13.019343+00:00 [queued]>
[2022-07-13 09:09:15,470] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-13 09:09:15,472] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-07-13 09:09:15,473] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-13 09:09:15,506] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-07-13 09:09:13.019343+00:00
[2022-07-13 09:09:15,525] {standard_task_runner.py:52} INFO - Started process 1529 to run task
[2022-07-13 09:09:15,532] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'second_task', 'manual__2022-07-13T09:09:13.019343+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpnb8m2b0z', '--error-file', '/tmp/tmp40f2ify2']
[2022-07-13 09:09:15,546] {standard_task_runner.py:80} INFO - Job 12: Subtask second_task
[2022-07-13 09:09:15,717] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v3.second_task manual__2022-07-13T09:09:13.019343+00:00 [running]> on host 8e2218deeaf2
[2022-07-13 09:09:15,850] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=harshab
AIRFLOW_CTX_DAG_ID=our_first_dag_v3
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-07-13T09:09:13.019343+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-13T09:09:13.019343+00:00
[2022-07-13 09:09:15,854] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-13 09:09:15,857] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo I am the second task']
[2022-07-13 09:09:15,879] {subprocess.py:85} INFO - Output:
[2022-07-13 09:09:15,885] {subprocess.py:92} INFO - I am the second task
[2022-07-13 09:09:15,887] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-13 09:09:15,944] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=second_task, execution_date=20220713T090913, start_date=20220713T090915, end_date=20220713T090915
[2022-07-13 09:09:16,008] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-13 09:09:16,067] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
